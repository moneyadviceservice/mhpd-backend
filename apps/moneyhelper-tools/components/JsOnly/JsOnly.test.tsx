import { render } from '@testing-library/react';
import { JsOnly } from '.';

describe('JsOnly component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <JsOnly fallback={'fallback text'}>
        <ul>
          <li>one</li>
          <li>two</li>
          <li>three</li>
        </ul>
      </JsOnly>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
