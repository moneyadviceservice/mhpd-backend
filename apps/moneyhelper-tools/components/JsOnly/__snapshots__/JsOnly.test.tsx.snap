// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JsOnly component renders correctly 1`] = `
<ul>
  <li>
    one
  </li>
  <li>
    two
  </li>
  <li>
    three
  </li>
</ul>
`;
